src/PlaywrightEnvironment.js

```diff
@@ -45,6 +46,10 @@ async function getBrowserPerProcess(playwrightInstance, config) {
     const browserType = getBrowserType(config)
     checkBrowserEnv(browserType)
     const { launchBrowserApp } = config
+    // https://github.com/mmarkelov/jest-playwright/issues/42#issuecomment-589170220
+    if (browserType !== CHROMIUM && launchBrowserApp.args) {
+      launchBrowserApp.args = []
```

Comment:
I would only remove the `--no-sandbox` argument by filtering it out.

Reply:
@mxschmitt yeah! That makes sense